pwiz_tools/Skyline/Model/GlobalizedObject.cs

```diff
@@ -156,12 +164,35 @@ public override int GetHashCode()
         }
 
         public string Serialize()
+        {
+            StringBuilder sb = new StringBuilder();
+            StringWriter sw = new StringWriter(sb);
```

Comment:
You don't need the separate StringBuilder here variable here. You can just call the zero-argument StringWriter constructor and return stringWriter.ToString() at the end.